server/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryDemoIT.java

```diff
@@ -483,4 +483,147 @@ public void InTest() throws ClassNotFoundException {
     }
     return actualIndexToExpectedIndexList;
   }
+
+  @Test
+  public void LikeTest() throws ClassNotFoundException {
+    String[] retArray =
+        new String[] {
+          "1509465600000,v2,true,", "1509465660000,v2,true,", "1509465720000,v1,false,",
+        };
+
+    Class.forName(Config.JDBC_DRIVER_NAME);
```

Comment:
It's better to split this test into three single test. It will be easier to debug and maintain.

Reply:
I split this test into two single test,One can find the result, the other can't find the result